srcpkgs/python3-mpv/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'python3-mpv'
 pkgname=python3-mpv
-version=1.0.3
-revision=2
+version=1.0.6
+revision=1
 build_style=python3-pep517
 # this test takes too long and has a low chance of failure
 # https://github.com/jaseg/python-mpv/issues/209#issuecomment-1180248112
-make_check_args="--deselect tests/test_mpv.py::TestLifecycle::test_wait_for_prooperty_event_overflow"
+make_check_args="-v --deselect tests/test_mpv.py::TestLifecycle::test_wait_for_property_event_overflow"
```

Comment:
The package appears to have issues with at least one test hanging and never finishing, without verbose it's difficult to determine which one it is since it only outputs periods, one for each finished test. Is that enough to justify enabling the flag?

Reply:
Yes, that's sufficient to justify enabling verbose mode.

Reply:
Alright, thank you!